static/js/people.js

```diff
@@ -297,7 +305,8 @@ exports.get_recipients = function (user_ids_string) {
     // See message_store.get_pm_full_names() for a similar function.
 
     const user_ids = user_ids_string.split(',');
-    const other_ids = _.reject(user_ids, exports.is_my_user_id);
+    let other_ids = _.reject(user_ids, exports.is_my_user_id);
+    other_ids = map_int(other_ids);
```

Comment:
This conversion should be moved up to `const user_ids = map_int(user_ids_string.split(','))`.